src/content/app/genome-browser/components/track-settings-panel/track-settings-views/components/track-settings-radio-options/TrackSettingsRadioOptions.tsx

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Do we still call it TrackSettingsRadioOptions? Didn't we decide to rename it to something like GlobalTrackSwitcher?

Reply:
yes, `GlobalTrackSwitch`